src/vs/editor/common/controller/cursorDeleteOperations.ts

```diff
@@ -49,7 +49,8 @@ export class DeleteOperations {
 	}
 
 	private static _isAutoClosingPairDelete(config: CursorConfiguration, model: ICursorSimpleModel, selections: Selection[]): boolean {
-		if (!config.autoClosingBrackets) {
+		if (!(config.autoClosingBrackets.autoClose || config.autoClosingBrackets.autoWrap
+			|| config.autoClosingQuotes.autoClose || config.autoClosingQuotes.autoWrap)) {
```

Comment:
May you create a small private function which will contain this check? It will be better for readability.